demos/wascc/fileserver/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+#[macro_use]
+extern crate log;
+extern crate wascc_actor as actor;
+extern crate wascc_codec;
+
+use actor::prelude::*;
+
+use wascc_codec::blobstore::Blob;
+use wascc_codec::serialize;
+
+actor_handlers! {
+    codec::http::OP_HANDLE_REQUEST => fetch,
+    codec::core::OP_HEALTH_REQUEST => health
+}
+
+fn fetch(r: codec::http::Request) -> CallResult {
+    // k8s volumes are mounted into the waSCC runtime using the same volume mount name
+    let store = objectstore::host("storage");
+    let mut path = String::from(r.path);
+
+    // strip the leading slash from the path
+    path = path.replacen("/", "", 1);
```

Comment:
This should probably use `trim_start_matches` in case for some reason the path doesn't start with a `/`